src/images/search.svg

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Created with Inkscape (http://www.inkscape.org/) -->
+
+<svg
+   width="66.792915mm"
+   height="66.104195mm"
+   viewBox="0 0 66.792915 66.104195"
+   version="1.1"
+   id="svg5"
+   sodipodi:docname="search-icon.svg"
+   inkscape:version="1.1 (c68e22c387, 2021-05-23)"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:svg="http://www.w3.org/2000/svg">
+  <sodipodi:namedview
+     id="namedview7"
+     pagecolor="#505050"
+     bordercolor="#eeeeee"
+     borderopacity="1"
+     inkscape:pageshadow="0"
+     inkscape:pageopacity="0"
+     inkscape:pagecheckerboard="0"
+     inkscape:document-units="mm"
+     showgrid="true"
+     showguides="true"
+     inkscape:guide-bbox="true"
+     inkscape:snap-global="false"
```

Comment:
this is pretty nasty, bloated SVG code making this file unnecessarily big and it also has a lot of inline styles overrides. Could you please run this either through Figma, or use our search icon from market of from assets file https://www.figma.com/file/UXKjbq8bNSGWHkHh2UPIsM/Assets?node-id=0%3A3851